packages/main/src/Tokenizer.ts

```diff
@@ -465,35 +476,34 @@ class Tokenizer extends UI5Element {
 		tokens.forEach(token => { token.selected = !tokensAreSelected; });
 	}
 
-	_handleTokenSelection(event, deselectAll = true) {
-		if (event.target.hasAttribute("ui5-token")) {
-			const deselectTokens = deselectAll ? this._tokens : [event.target];
+	_handleTokenSelection(e: KeyboardEvent | MouseEvent, deselectAll = true) {
+		const target = e.target as Token;
+		if (target.hasAttribute("ui5-token")) {
+			const deselectTokens = deselectAll ? this._tokens : [e.target];
```

Comment:
use : `const deselectTokens = deselectAll ? this._tokens : []; `. typescript understands that this could be array of tokens. then the casting inside the forEach is redudant

Reply:
Done